charts/azuremonitor-containers/templates/ama-logs-openshift-scc.yaml

```diff
@@ -0,0 +1,27 @@
+{{- if eq .Values.Azure.Cluster.Distribution "openshift" }}
+apiVersion: security.openshift.io/v1
+kind: SecurityContextConstraints
+metadata:
+  name: ama-logs-ds-scc
```

Comment:
SCC required for both daemonset and replicaset, right? then in that case, make the name change from "ama-logs-ds-scc" to "ama-logs-scc"